tests/integration/components/o-s-s/attribute-text-test.ts

```diff
@@ -0,0 +1,95 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+import click from '@ember/test-helpers/dom/click';
+import triggerEvent from '@ember/test-helpers/dom/trigger-event';
+
+module('Integration | Component | o-s-s/attribute-text', function (hooks) {
+  setupRenderingTest(hooks);
+
+  module('Default behavior', function () {
+    module('Render conditions', function () {
+      test('it renders', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @value="World" />`);
+        assert.dom('.oss-attribute').exists();
+        assert.dom('.oss-attribute__label').hasText('Hello');
+        assert.dom('.oss-attribute__value').hasText('World');
+      });
+
+      test('It renders when the @value is undefined', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" />`);
+        assert.dom('.oss-attribute').exists();
+        assert.dom('.oss-attribute__value').hasText('-');
+      });
+
+      test('it renders with a tooltip when specified', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @tooltip="Tooltip" />`);
+        assert.dom('.oss-attribute__info').exists();
+        await assert.tooltip('.oss-attribute__info').hasTitle('Tooltip');
+      });
+
+      test('it renders a dash as value when no value is specified', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" />`);
+        assert.dom('.oss-attribute').exists();
+        assert.dom('.oss-attribute__value').hasText('-');
+      });
+
+      test('it renders a dash as value when @value is empty', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @value=""/>`);
+        assert.dom('.oss-attribute').exists();
+        assert.dom('.oss-attribute__value').hasText('-');
+      });
+    });
+
+    module('Copy action', function (hooks) {
+      hooks.beforeEach(function () {
+        this.textForCopy = 'copied value';
+        this.displayCopyBtn = true;
+      });
+
+      test('the text is copyable by default', async function (assert) {
+        await render(hbs`<OSS::Attribute::Text @label="Hello" @value={{this.textForCopy}} />`);
+
+        await triggerEvent('.oss-attribute', 'mouseenter');
```

Comment:
I wanted to test that as well but the read in multiple locations from the ember community that `mouseenter` & `mouseleave` were unsafe events and thus didn't worked with `triggerEvent`. IT actually works ? 😱 
I'll be updating the Attribute::Country later then 👍